File: GanttView/RadGanttViewIndicatingSpecialDays/RadGanttViewIndicatingSpecialDaysVB/CustomGanttView.vb
Imports Telerik.WinControls.UI

Public Class CustomGanttView
    Inherits RadGanttView

    Protected Overrides Function CreateGanttViewElement() As RadGanttViewElement
        Return New CustomGanttViewElement()
    End Function

    Public Overrides Property ThemeClassName As String
        Get
            Return GetType(RadGanttView).FullName
        End Get
        Set(value As String)
            MyBase.ThemeClassName = value
        End Set
    End Property
End Class

File: GanttView/RadGanttViewIndicatingSpecialDays/RadGanttViewIndicatingSpecialDays/CustomGanttView.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls.UI;

namespace RadGanttViewIndicatingSpecialDays
{
    public class CustomGanttView : RadGanttView
    {
        protected override RadGanttViewElement CreateGanttViewElement()
        {
            return new CustomGanttViewElement();
        }

        public override string ThemeClassName
        {
            get { return typeof(RadGanttView).FullName; }
        }
    }
}
